src/main/java/com/cleancommunity/user/User.java

```diff
@@ -9,6 +9,7 @@
 public abstract class User {
 
     public static PostingDAO postingDAO = new MysqlPostingDAO();
+    public static UserDAO userDAO = new MysqlUserDAO();
```

Comment:
Do we still want this? I think our app is small enough that we can just have the DAOs in the Controllers. @JakeC1020 @ryan-turner 

This way, we can depend on Spring to serialize our class into JSON by just returning the User/Posting classes. 

If we continue this route with the User containing the DAO, I don't know of a Spring annotation to prevent a function's value from being returned and we'd have to write our own JSON serializer. 

Either way should be fine since there are so implementations for this.